setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(name='rac-aspace',
+      version='0.0.1',
+      description='Helpers for using the ArchivesSpace API using ArchivesSnake',
+      long_description=long_description,
```

Comment:
```suggestion
      long_description=long_description,
```
This seems wrong? Shouldn't there actually be some description here, otherwise we're just setting the value of the variable to itself...

Reply:
From my understanding of the following article, it should add the readme to the package description: https://packaging.python.org/tutorials/packaging-projects/

`long_description=long_description` is making the Readme file into the long description added to the package.

`long_description is a detailed description of the package. This is shown on the package detail package on the Python Package Index. In this case, the long description is loaded from README.md which is a common pattern.`

Reply:
You're right - I was focused on the setup block and missed that you'd opened that file and read the contents into `long_description`.